### ISSUE TITLE
Replace integers with bools in Conch's cipherMap

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6807]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#6807|
|Type|enhancement|
|Created|2013-11-01 00:09:03Z|
|Branch|https://github.com/twisted/twisted/tree/conch-bools-6807|

This makes it more readable.

Attachments:

* [t6807.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/e25/e25209a9755e4093686a2d7966cf519aa825607e/8a594856f449dfd471547a7955e04f959f2fa705.diff) (1834 bytes) - added by Alex on 2013-11-01 00:09:18Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6807 6807
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_conch_bools_6807 branches/conch-bools-6807
branch_author__lvh lvh
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1383264543000000 1383264543000000
changetime__1383387793000000 1383387793000000
version__None None
owner__lvh lvh
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_2"></a>|@lvh set owner to @lvh|
|-|-|

Yes.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_3"></a>|@lvh commented|
|-|-|

(In [40547]) Branching to 'conch-bools-6807'

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_4"></a>|@lvh set status to `closed`|
|-|-|

(In [40550]) Merge conch-bools-6807: Use bools instead of 0, 1 in conch's SSH cipher specifications

Author: Alex
Reviewer: lvh
Fixes: [#6807](https://github.com/twisted/twisted/issues/6807)

Conch previously used 0 and 1 instead of False and True in an internal data structure. The formatting of said data structure was also a little off. This fixes both, improving the legibility of the code a little.
